### PR TITLE
added resources for gate notes.

### PR DESCRIPTION
**Pull request by Yathartha22**
I have added another useful resource of GATE notes(even I am currently studying them and found them really useful).
Also a slight typo fix(INstitute --> Institute)

### CODE REVIEW

File: GATE/GateNotes.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # GATE Notes #
 
-* Here is the list of GATE Notes.You can find latest notes according to latest Syllabus.Its notes of Coaching INstitute Gate Made Easy. So [Here is the Link](https://drive.google.com/open?id=0B47CU_twbDB6fnBMUzhfeXV4UGlETmdnOFE1ajItdlBDWF95SHFrVWN0bmMwaVR0SmIwY3M)
+* Here is the list of GATE Notes.You can find latest notes according to latest Syllabus.Its notes of Coaching Institute Gate Made Easy. So [Here is the Link](https://drive.google.com/open?id=0B47CU_twbDB6fnBMUzhfeXV4UGlETmdnOFE1ajItdlBDWF95SHFrVWN0bmMwaVR0SmIwY3M)
```
Comment (vicky002) [inline GATE/GateNotes.md]: There should be a space between `Syllabus.Its` , Please fix it. 

`Here is the list of GATE Notes` should be used once. Make it as a small `h3` headline and write all these links **Under the headline** as bullet points.

**Review by vicky002 — DISMISSED** [review]

**Comment by Yathartha22** [comment]
Have a look now.

**Comment by Yathartha22** [comment]
I guess this looks good now @vicky002 .

**Review by Byte7 — APPROVED** [review]
Thanks @Yathartha22 for the changes. Looks Slick!

**Pull request merged**